tests/page/page-emulate-media.spec.ts

```diff
@@ -114,3 +114,15 @@ it('should change the actual colors in css', async ({page}) => {
   await page.emulateMedia({ colorScheme: 'light' });
   expect(await getBackgroundColor()).toBe('rgb(255, 255, 255)');
 });
+
+it('should emulate reduced motion', async ({page}) => {
```

Comment:
Let's have a test for default behavior, if we decide to not fallback to system default.

Reply:
how shall we test it?

Reply:
Just match `no-preference` before the first `emulateMedia` call.